edxval/api.py

```diff
@@ -561,7 +581,12 @@ def import_from_xml(xml, edx_video_id, course_id=None):
             course_id,
         )
         if course_id:
-            CourseVideo.get_or_create_with_validation(video=video, course_id=course_id)
+            course_video, __ = CourseVideo.get_or_create_with_validation(video=video, course_id=course_id)
+
+            image_file_name = xml.get('image', '').strip()
+            if image_file_name:
+                VideoImage.create_or_update(course_video, image_file_name)
```

Comment:
Same as above, Do we need atomic transaction covering creation of `CourseVideo` and `VideoImage` here?